cmd/explaintest/r/subquery.result

```diff
@@ -55,13 +55,11 @@ insert into exam values(1, 'math', 100);
 set names utf8 collate utf8_general_ci;
 explain format = 'brief' select * from stu where stu.name not in (select 'guo' from exam where exam.stu_id = stu.id);
 id	estRows	task	access object	operator info
-Apply	10000.00	root		CARTESIAN anti semi join, other cond:eq(test.stu.name, Column#8)
+HashJoin	8000.00	root		anti semi join, equal:[eq(test.stu.id, test.exam.stu_id)], other cond:eq(test.stu.name, "guo")
```

Comment:
The change is correct the NOT IN's key name not in('guo') is put in other cond. And the inner filter `exam.stu_id = stu.id` is put at the join key.